packages/reports/addon/components/dropdown-date-picker.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2018, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * Usage:
+ *   {{dropdown-date-picker
+ *       onApply=(action 'onApply')
```

Comment:
`onApply`? or `onSelect`?

Reply:
It's called when the apply button is clicked